guides/micronaut-security-jwt/kotlin/src/main/kotlin/example/micronaut/HomeController.kt

```diff
@@ -1,19 +1,18 @@
 package example.micronaut
 
-import io.micronaut.http.MediaType
+import io.micronaut.http.MediaType.TEXT_PLAIN
 import io.micronaut.http.annotation.Controller
 import io.micronaut.http.annotation.Get
 import io.micronaut.http.annotation.Produces
 import io.micronaut.security.annotation.Secured
+import io.micronaut.security.rules.SecurityRule
 import java.security.Principal
 
-@Secured("isAuthenticated()") // <1>
-@Controller("/") // <2>
+@Secured(SecurityRule.IS_AUTHENTICATED) // <1>
+@Controller // <2>
 class HomeController {
 
-    @Produces(MediaType.TEXT_PLAIN) // <3>
-    @Get("/")  // <4>
-    fun index(principal: Principal): String {  // <5>
-        return principal.name
-    }
+    @Produces(TEXT_PLAIN)
+    @Get // <3>
+    fun index(principal: Principal): String = principal.name // <4>
```

Comment:
we got rid of the <5> callout. Did we remove it from the asciidoc as well?

Reply:
The Kotlin code had the extra callout, the Java and Groovy have 4 and the asciidoc has 4